packages/trader/src/Modules/SmartChartBeta/Components/LaunchModal/launch-modal.tsx

```diff
@@ -0,0 +1,74 @@
+import { Button, DesktopWrapper, MobileWrapper, Modal, PageOverlay, UILoader } from '@deriv/components';
+import { LocalStore, getUrlBase } from '@deriv/shared';
+
+import { Localize } from '@deriv/translations';
+import React, { useState } from 'react';
+import './launch-modal.scss';
+import { observer } from 'mobx-react';
+import { useStore } from '@deriv/stores';
+
+const ContinueButton: React.FC<{ handleOpen: () => void }> = ({ handleOpen }) => (
+    <Modal.Footer>
+        <Button has_effect onClick={handleOpen} primary large>
+            <Localize i18n_default_text='Continue' />
+        </Button>
+    </Modal.Footer>
+);
+
+const InfoDisplay: React.FC = () => (
+    <div className='modal-content' data-testid='launch-modal'>
+        <img src={getUrlBase('/public/images/common/chart-launch.png')} alt='Chart Image' />
+        <h1 className='title'>
+            <Localize i18n_default_text='Deriv Trader Chart v2.0.' />
+        </h1>
+        <p className='sub-title'>
+            <Localize i18n_default_text='Smoother charts. Smarter insights.' />
+        </p>
```

Comment:
Please use Text component instead and pass font size and other text styles to it. You can use it as `p` or `h1` as well.